app/controllers/concerns/foreman/controller/registration_commands.rb

```diff
@@ -19,6 +19,14 @@ def insecure
     registration_params['insecure'] ? '--insecure' : ''
   end
 
+  def is_token_valid(min_value, max_value)
+    registration_params['jwt_expiration'].to_i.between?(min_value, max_value)
+  end
+
+  def unlimited
```

Comment:
```suggestion
  def unlimited?
```

or maybe even

```suggestion
  def expiration_unlimited?
```

Not sure.